cmd/prom2json/main_test.go

```diff
@@ -7,21 +7,22 @@ import (
 
 	"github.com/davecgh/go-spew/spew"
 	dto "github.com/prometheus/client_model/go"
+	p2j "github.com/ChristianKniep/prom2json"
```

Comment:
Also, please have a blank line in front. (We separate imports from the same repo.)